tests/utils/TestCase/DominantColorTestCase.php

```diff
@@ -13,107 +13,91 @@ abstract class DominantColorTestCase extends WP_UnitTestCase {
 			),
-			'green_gif'   => array(
+			'green_gif'     => array(
 				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/green.gif',
-				'expected_color' => array( '1ee204', '1ee24' ),
+				'expected_color' => array( '00ff00', '0ff0', 'ff01', 'ff00' ),
 				'is_wp_error'    => false,
 			),
-			'white_gif'   => array(
+			'white_gif'     => array(
 				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/white.gif',
 				'expected_color' => array( 'ffffff' ),
 				'is_wp_error'    => false,
 			),
-			'trans_gif'   => array(
+			'trans_gif'     => array(
 				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/trans.gif',
-				'expected_color' => array( 'ffffff' ),
-				'is_wp_error'    => false,
-			),
-			'trans4_gif'  => array(
-				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/trans4.gif',
-				'expected_color' => array( '133f00', '4cfc0' ),
+				'expected_color' => array( '5a5a5a', '20202' ),
 				'is_wp_error'    => false,
 			),
 
-			'red_png'     => array(
+			'red_png'       => array(
 				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/red.png',
-				'expected_color' => array( 'ff0505', 'ff55', 'ff0506', 'ff56' ),
+				'expected_color' => array( 'ff0000', 'ff00' ),
 				'is_wp_error'    => false,
 			),
-			'green_png'   => array(
+			'green_png'     => array(
 				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/green.png',
-				'expected_color' => array( '1ee204', '1ee24' ),
+				'expected_color' => array( '00ff00', '0ff0', 'ff00', 'ff01' ),
 				'is_wp_error'    => false,
 			),
-			'white_png'   => array(
+			'white_png'     => array(
 				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/white.png',
 				'expected_color' => array( 'ffffff' ),
 				'is_wp_error'    => false,
 			),
-
-			'trans4_png'  => array(
-				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/trans4.png',
-				'expected_color' => array( '133f00', '4cfc0' ),
+			'trans_png'     => array(
+				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/trans.png',
+				'expected_color' => array( '000' ),
 				'is_wp_error'    => false,
 			),
 
-			'red_webp'    => array(
+			'red_webp'      => array(
 				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/red.webp',
-				'expected_color' => array( 'ff0505', 'ff55', 'ff0506', 'ff56' ),
+				'expected_color' => array( 'ff0000', 'ff00' ),
 				'is_wp_error'    => false,
 			),
-			'green_webp'  => array(
+			'green_webp'    => array(
 				'image_path'     => TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/dominant-color/green.webp',
-				'expected_color' => array( '1de303', '1de33' ),
+				'expected_color' => array( '00ff00', '0ff0', 'ff01', 'ff00' ),
```

Comment:
Can we ensure that we ALWAYS generate a 6 character hex color? I don't love the results are this unpredictable here.